website/spam_admin/views.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+from modularodm import Q
+from framework.exceptions import HTTPError
+from website.models import Node
+from website.models import Comment
+from framework.auth.decorators import must_be_logged_in
+from .decorators import must_be_spam_admin
+from .utils import serialize_comments, serialize_projects, train_spam_project, train_spam
+from flask import request
+import logging
+import httplib as http
+from website.settings import SPAM_ASSASSIN as spam_assassin_active
+logger = logging.getLogger(__name__)
+
+################################### COMMENTS ############################################################
+@must_be_logged_in
+@must_be_spam_admin
+def init_spam_admin_comments_page(**kwargs):
+    """
+    determine whether user is on comments page or projects page
+    """
+    return {}
+
+@must_be_logged_in
+@must_be_spam_admin
+def list_comment_page(**kwargs):
+    """
+    make a list of comments that are marked as possibleSpam
+    """
+    if not spam_assassin_active:
+        return {
+            'comments': list([]),
```

Comment:
How will seeing an empty list of comment spam tell the spam_admin that the spam assassin server is not running?